package.json

```diff
@@ -112,6 +111,12 @@
           "npmPublish": false
         }
       ],
+      [
+        "@semantic-release/npm",
```

Comment:
It's just a name and it works with yarn too?

Reply:
Yes 😄. It is used to bump a version in `package.json` and publish it to NPM